### PR TITLE
Fix Stereochemistry of Phytanic Acid Oxidation

### PR DESCRIPTION
**Pull request by Devlin-Moyer**
#### Main improvements in this PR:

As proposed in #773:

- Renamed `MAM02746e`, `MAM02746c` and `MAM02746x` to "(3R)-phytanic acid"
- Brought back previously-deprecated `MAM03884e`, `MAM03884c` and `MAM03884x` and renamed them to "(3S)-phytanic acid"
- Brought back previously-deprecated `MAR00659` and `MAR01617` (exchange and [e] <-> [c] transport reactions for `MAM03884`)
- Renamed `MAM02747c` and `MAM02747x` to "(3R)-phytanoyl-CoA"
- Replaced `MAM02746c` with `MAM03884c` in `MAR03388`
- Edited `MAR03481` to be `MAM02747c + MAM01371c + 2 MAM02040c -> MAM02746x + MAM01597c + MAM02039c + MAM01285c + MAM02751c`, GPR: `ENSG00000117528`, references: `PMID:24333844;PMID:33500543`
- Edited `MAR06779` to be `MAM03362c + MAM01371c + 2 MAM02040c -> MAM03384x + MAM01597c + MAM02039c + MAM01285c + MAM02751c`, GPR: `ENSG00000117528`, references: `PMID:24333844;PMID:33500543`
- Replaced `MAM02746x` with `MAM03884x` in `MAR03389`
- Renamed `MAM00655x` to "(3R)-2-hydroxyphytanoyl-CoA"
- Created `MAM20083x` to represent peroxisomal (2R)-pristanal
- Replaced (2S)-pristanal (`MAM00564x`) with `MAM20083x` in `MAR03486`
- Renamed `MAM02766e`, `MAM02766c`, and `MAM02766x` to "(2S)-pristanic acid"
- Brought back previously-deprecated `MAR03488` and `MAM00077x`
- Replaced `MAM00564x` with `MAM20083x` in `MAR03488`
- Replaced `MAM02766x` with `MAM00077x` in `MAR03493`
- Created `MAM00077c` and `MAM00077e` to represent cytosolic and extracellular (2R)-pristanic acid
- Replaced `MAM02766c` with `MAM00077c` in `MAR03390`
- Created `MAR20179` to represent ATP-dependent export of (2R)-pristanic acid from the cytosol to the extracellular environment
- Created `MAR20180` to represent exchange of (2R)-pristanic acid

While I was creating `MAM00077c` and `MAM00077e`, I noticed that the `MAM02766` metabolites were associated with both `MNXM3342` and `MNXM7698` in `metabolites.tsv`, but `MNXM7698` has apparently been deprecated in favor of `MNXM3422`, so I edited those rows so that they're only associated with `MNXM3342`.

**I hereby confirm that I have:**
- [ ] Tested my code on my own computer for running the model
- [X] Selected `develop` as a target branch
- [X] Any removed reactions and metabolites have been moved to the corresponding deprecated identifier lists

### CODE REVIEW

File: model/Human-GEM.yml [hunk 1]
```diff
@@ -117602,15 +117658,21 @@
       - id: "MAR03481"
       - name: ""
       - metabolites: !!omap
+          - MAM01285c: 1
+          - MAM01371c: -1
+          - MAM01597c: 1
+          - MAM02039c: 1
+          - MAM02040c: -2
```
Comment (JHL-452b) [inline model/Human-GEM.yml:117665]: > Edited MAR03481 to be MAM02747c + MAM01371c + 2 MAM02040m -> MAM02746x + MAM01597c + MAM02039c + MAM01285c + MAM02751c, GPR: ENSG00000117528, references: PMID:24333844;PMID:33500543

There is a discrepancy between the description in PR and changed file,  `MAM02040m` was set as reactant in PR but `MAM02040c` in the file.  May be it's necessary to check which one is correct.

File: model/Human-GEM.yml [hunk 2]
```diff
@@ -242479,18 +242592,19 @@
       - metabolites: !!omap
           - MAM01285c: 1
           - MAM01371c: -1
+          - MAM01597c: 1
           - MAM02039c: 1
-          - MAM02040c: -1
-          - MAM02747c: -1
-          - MAM02747x: 1
+          - MAM02040c: -2
```
Comment (JHL-452b) [inline model/Human-GEM.yml:242597]: Same problem as above comments.

File: model/Human-GEM.yml (hunk 1 above)
Comment (Devlin-Moyer) [reply model/Human-GEM.yml:117665]: ah yea it was supposed to be `MAM02040c`; I'm not sure why I had `MAM02040m` in the issue and PR, but I've updated both of those to match what I actually wrote in `model/Human-GEM.yml`

File: model/Human-GEM.yml (hunk 2 above)
Comment (Devlin-Moyer) [reply model/Human-GEM.yml:242597]: ah yea it was supposed to be `MAM02040c`; I'm not sure why I had `MAM02040m` in the issue and PR, but I've updated both of those to match what I actually wrote in `model/Human-GEM.yml`

**Review by feiranl — APPROVED** [review]
LGTM!

**Pull request merged**